python/lsst/pipe/supertask/activator.py

```diff
@@ -407,6 +447,15 @@ def parse_and_run(cls):
             for package in args.packages:
                 TASK_PACKAGES[package] = importlib.import_module(package)
 
+        if args.package_paths:
+            TASK_PACKAGES.clear()
+            for package_path in args.package_paths:
+                if not os.path.exists(package_path):
+                    print(package_path, ' does not exist!')
```

Comment:
Do you mean to continue to create the mock class below even though we know the path is broken?